RecoTauTag/HLTProducers/src/CaloTowerFromL1TSeededCreatorForTauHLT.cc

```diff
@@ -0,0 +1,98 @@
+// makes CaloTowerCandidates from CaloTowers
+// original author: L.Lista INFN, modifyed by: F.Ratnikov UMd 
+// Author for regionality A. Nikitenko
+// Modified by S. Gennai + T. Strebler
+
+#include "DataFormats/RecoCandidate/interface/RecoCaloTowerCandidate.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "RecoTauTag/HLTProducers/interface/CaloTowerFromL1TSeededCreatorForTauHLT.h"
+// Math
+#include "Math/GenVector/VectorUtil.h"
+#include <cmath>
+
+using namespace edm ;
+using namespace reco;
+using namespace std ;
+
+CaloTowerFromL1TSeededCreatorForTauHLT::CaloTowerFromL1TSeededCreatorForTauHLT( const ParameterSet & p ) 
+  :
+  mVerbose                                            (p.getUntrackedParameter<int> ("verbose"        , 0) ),
+  mtowers_token     (consumes<CaloTowerCollection>    (p.getParameter<InputTag>     ("towers"            ))),
+  mCone                                               (p.getParameter<double>       ("UseTowersInCone"   ) ),
+  mTauTrigger_token (consumes<trigger::TriggerFilterObjectWithRefs>   (p.getParameter<InputTag>     ("TauTrigger"        ))),
+  mEtThreshold                                        (p.getParameter<double>       ("minimumEt"         ) ),
+  mEThreshold                                         (p.getParameter<double>       ("minimumE"          ) )
+{
+  produces<CaloTowerCollection>();
+}
+
+CaloTowerFromL1TSeededCreatorForTauHLT::~CaloTowerFromL1TSeededCreatorForTauHLT() {
```

Comment:
please replace this with
`CaloTowerFromL1TSeededCreatorForTauHLT::~CaloTowerFromL1TSeededCreatorForTauHLT() = default;`